generator/generator.go

```diff
@@ -336,7 +336,11 @@ func (g *generator) writeTemplate(t parser.HTMLTemplate) error {
 		}
 		{
 			indentLevel++
-			// _, err = io.Copy(w, buf)
+			// templ.ReleaseBuffer(templBuffer)
```

Comment:
Shouldn't the buffer be released _after_ copying the results?

Reply:
Yes, definitely.
Updated that.